inferno/run-inferno.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# Get Config
+CONFIG=`cat config.json`
+BULK_URL=$(echo $CONFIG | jq -r ".bulk_url")
+TOKEN_URL=$(echo $CONFIG | jq -r ".auth.token_endpoint")
+CLIENT_ID=$(echo $CONFIG | jq -r ".auth.client_id")
+CLIENT_SECRET=$(echo $CONFIG | jq -r ".auth.client_secret")
+TESTS=$(echo $CONFIG | jq -r ".required_tests.[]")
+
+if [[ "$CLIENT_ID" == "" ]]; then
+  echo "
+  client_id not set in config. exiting...
+  "
+  exit 1
+fi
+
+if [[ "$CLIENT_SECRET" == "" ]]; then
+  echo "
+  client_secret not set in config. exiting...
+  "
+  exit 1
+fi
+
+### TODO: Cloning the bulk test repo, running the docker compose/build, and
+### breaking things down at the end will all be handled outside of this script.
+### As part of automating the tests, we will remove these steps from here. This
+### is just for making running the script locally a little easier.
+
+# Clone test kit repo
+if [ ! -d "bulk-data-test-kit" ] ; then
+  git clone https://github.com/inferno-framework/bulk-data-test-kit.git
+fi
+
+cd bulk-data-test-kit
+git checkout 957190d889b544d573766d1c1178da22496ce62a
+
+# Build / Compose Inferno Docker container
+./setup.sh
+docker compose build
+docker compose up -d
+sleep 10
+
+# Stop the hl7 validator one - it takes a lot of time / memory to run, and those tests are not applicable
+docker stop bulk-data-test-kit-hl7_validator_service-1
```

Comment:
If we know what services we want, we can specify them and combine a few steps here.
```suggestion
docker compose up --build -d inferno worker nginx redis
```
```